packages/scandipwa/src/route/ConfirmAccountPage/ConfirmAccountPage.style.scss

```diff
@@ -62,6 +62,17 @@
         }
     }
 
+    &-WarningMsg {
+        padding-top: 20px;
+        text-align: center;
+        color: var(--primary-error-color);
+        font-size: 1.2em;
```

Comment:
Can we change this to absolute unit px instead of em? Just to be consistent.

Reply:
Done.